src/Model/Subscription.php

```diff
@@ -12,7 +12,7 @@
 use Friendica\Database\DBA;
 use Friendica\DI;
 use Friendica\Factory\Api\Mastodon\Notification as NotificationFactory;
-use Friendica\Navigation\Notifications\Entity;
+use Friendica\Navigation\Notifications\Entity\Notification as EntityNotification;
```

Comment:
Please rename to `NotificationEntity`.
```suggestion
use Friendica\Navigation\Notifications\Entity\Notification as NotificationEntity;
```

Reply:
Done.